packages/web/services/institutions.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable import/prefer-default-export */
+import { apiGet, apiPost } from './api';
+
+export const getInstitutions = async () => {
+	const response = await apiGet('institutions');
+
+	const { data } = response;
+	return data;
+};
+
+export const createInstitutions = async (payload) => {
+	const response = await apiPost('institutions', payload);
+
+	const { data } = response;
+	return data;
```

Comment:
Its important to safeguard this code against bad api response, as we do in other requests

Reply:
I second this! :arrow_up: 

I also missed those comments that keep it all documented like this:
![image](https://user-images.githubusercontent.com/10481028/101629857-be2d8f80-3a00-11eb-8ef6-6090f122b28d.png)

Reply:
I second this! :arrow_up: 

I also missed those comments that keep it all documented like this:
![image](https://user-images.githubusercontent.com/10481028/101629857-be2d8f80-3a00-11eb-8ef6-6090f122b28d.png)